packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/__tests__/quanticCategoryFacet.test.js

```diff
@@ -102,8 +124,12 @@ function cleanup() {
 }
 
 describe('c-quantic-category-facet', () => {
+  let consoleError;
+
   beforeAll(() => {
+    consoleError = jest.spyOn(console, 'error').mockImplementation(() => {});
```

Comment:
I would only mock the `console.error` for the specific tests that are expected to spy on `console.error` otherwise you're silencing the `console.error` potentially of all other tests with the `.mockImplementation` no?

I would move that mock specifically under the `beforeAll` of the `describe` of the tests that test the validation of the `dependsOn` instead of having this under the top level `describe`.